File: Java/dependency-injection/bastard-injection/src/main/java/com/github/grzesiek_galezowski/BastardInjection/Outbound/TcpSocket.java
package com.github.grzesiek_galezowski.BastardInjection.Outbound;

class TcpSocket implements IOutputSocket {
  public void open() {
    System.out.println("open");
  }
  public void close() {
    System.out.println("closing");
  }
  public void send(String lol) {
    System.out.println(lol);
  }
}

File: CSharp/DependencyInjection/DependencyInjection/ConformingContainerAntipattern/_3_ConformingContainer/Outbound/TcpSocket.cs
using System;

namespace ConformingContainerAntipattern._3_ConformingContainer.Outbound
{
  public interface IOutputSocket
  {
    void Open();
    void Close();
    void Send(string lol);
  }

  class TcpSocket : IOutputSocket
  {
    public void Open()
    {
      Console.WriteLine("open");
    }

    public void Close()
    {
      Console.WriteLine("closing");
    }

    public void Send(string lol)
    {
      Console.WriteLine(lol);
    }
  }
}